package.json

```diff
@@ -18,10 +18,13 @@
     "eslint-plugin-react": "^7.28.0",
     "istanbul": "^0.4.5",
     "mocha": "^9.1.4",
-    "serverless": "^2.72.1"
+    "serverless": "^2.72.1",
+    "aws-sdk": "^2.1061.0",
+    "randomstring": "^1.2.2",
+    "shelljs": "^0.8.5"
   },
   "scripts": {
-    "test": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- -R spec && node ./node_modules/istanbul/lib/cli.js check-coverage --line 70 coverage/coverage.json",
+    "test": "node ./node_modules/istanbul/lib/cli.js cover ./node_modules/mocha/bin/_mocha -- --recursive -R spec && node ./node_modules/istanbul/lib/cli.js check-coverage --line 70 coverage/coverage.json",
```

Comment:
Can you make a separate `integration-test` command for running just the integration tests. `test` command would be used for just unit tests